docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,23 +1,63 @@
 = Solidity Loader
 
- * Allows importing .sol files directly into source code.
- * Allows to specify contract name for .sol files, like require("Contract.sol?contract=Counter").
- * Tracks dependencies for contracts, so children are updated when parents are modified.
- * Converts .sol files into .json using `compile oz`, `oz push`, and `oz update`.
- * Uses only development network so won't run on any other networks.
- * You can set the development network name using the loader's config.
- * Handles race conditions at Webpack pipeline using a lock.
- * Pulls build directory from truffle config.
- * In case oz commands are not available, uses cached versions of .json files.
- * Reports errors to console output and dev tools.
- * Compilation can be disabled using the loader's disabled config.
+Solidity Hot Loader is a webpack loader that watches the smart contract `.sol` files you’re using. Once there’s a change, it compiles them, updates the local blockchain using the https://openzeppelin.com/sdk/[OpenZeppelin SDK], and refreshes your app in the browser while preserving your blockchain state. Compare this against the existing Solidity development cycle.
+
+The Solidity Hot Loader development cycle:
```

Comment:
I'd swap these two around, I think it looks more impressive that way

Reply:
Swapped